domain/src/main/kotlin/no/nav/su/se/bakover/domain/SøknadInnhold.kt

```diff
@@ -37,7 +37,7 @@ data class Oppholdstillatelse(
     val statsborgerskapAndreLandFritekst: String? = null
 ) {
     enum class OppholdstillatelseType() {
-        MIDLERTIG,
+        MIDLERTIDIG,
```

Comment:
Har du sjekket om dett har noen konsekvenser for eksisterende data?.. Finner 6 rakkere i basen med "MIDLERTIG" i sin søknad - bør sikkert slenge med et lite rename-script for basen slik at disse overlever.

Reply:
Bare dra med deg noen for å skrive et migreringsscript hvis du ikke har gjort det før, @RamziAbuQassim